src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -239,6 +239,12 @@
                     [value]="zaak.einddatum | datum"
                   >
                   </zac-static-text>
+                  <zac-static-text
+                    *ngIf="zaak.startdatumBewaartermijn"
+                    [label]="'startdatumBewaartermijn' | translate"
```

Comment:
no need to be a dynamic value, and the component itself translates it.

```suggestion
                    label="startdatumBewaartermijn"
```